qemu/tests/cfg/uefi_check_boot_option.cfg

```diff
@@ -0,0 +1,34 @@
+- uefi_check_boot_option:
+    only q35
+    only ovmf
+    type = uefi_check_boot_option
+    timeout = 360
+    start_vm = no
+    boot_menu = on
+    ovmf_log = "ovmf.log"
+    extra_params += " -global isa-debugcon.iobase=0x402 -debugcon file:%s"
+    variants:
+        - time_10:
+            extra_params += " -boot menu=on,splash-time=10000"
+            check_info_cmd = "grep PlatformBootManagerBeforeConsole %s"
+            expected_result = "10"
+        - time_12:
+            extra_params += " -boot menu=on,splash-time=12000"
+            check_info_cmd = "grep PlatformBootManagerBeforeConsole %s"
+            expected_result = "12"
+        - bootindex:
+            extra_params += " -boot menu=on,splash-time=12000"
+            check_info_cmd = "grep PlatformBootManagerBeforeConsole %s"
+            expected_result = "12"
+            variants:
+                - hard_disk:
+                    extra_params += " -drive file=/usr/share/edk2/ovmf/UefiShell.iso,if=none,cache=none,snapshot=off,aio=native,media=cdrom,id=cdrom1"
```

Comment:
please refer to case uefishell for adding uefishell.iso.  this is not recommended.

Reply:
OK,I will check it.
Thanks for your suggestion!